lib/jetstream/api/object.ex

```diff
@@ -0,0 +1,300 @@
+defmodule Jetstream.API.Object do
+  @moduledoc """
+  API for interacting with the JetStream Object Store
+
+  Learn more about Object Store: https://docs.nats.io/nats-concepts/jetstream/obj_store
+  """
+  alias Jetstream.API.{Consumer, Stream, Util}
+  alias Jetstream.API.Object.Meta
+
+  @stream_prefix "OBJ_"
+  @subject_prefix "$O."
+
+  def create_bucket(conn, bucket_name, params \\ []) do
```

Comment:
I could see the case for making an `Object.Bucket` module to separate responsibilities. I think from an interface perspective I would still want to delegate so calling `Object.create_bucket` is the public interface.

Reply:
👍 I can get behind that. I'll keep `create_bucket` and `delete_bucket` as-is. And Then change the rest of the functions to skip the `_object` since that focus of this API is to work with the files in a bucket.